test/e2e/operator_test.go

```diff
@@ -350,3 +354,106 @@ func customResolver(nameserver string) *net.Resolver {
 		},
 	}
 }
+
+// Test to verify the ExternalDNS should create the CNAME and txt record for the OpenshiftRoute
```

Comment:
This test doesn't test the creation of TXT record.

Reply:
taken care